specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/examples/PrivateEndpointCreate.json

```diff
@@ -0,0 +1,96 @@
+{
+  "parameters" : {
+    "api-version": "2019-04-01",
+    "subscriptionId" : "subId",
+    "resourceGroupName": "rg1",
+    "privateEndpointName": "testPe",
+    "parameters": {
+      "properties": {
+        "privateLinkServiceConnections": [
+          {
+            "properties": {
+              "privateLinkServiceId":"/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/privateLinkServices/testPls",
+              "groupIds": [
+                "groupIdFromResource"
+              ],
+              "requestMessage": "Please approve my connection."
+            }
+          }
+        ],
+        "subnet": {
+          "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/myVnet/subnets/mySubnet"
+        }
+      }
+    }
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "name" : "testPe",
+        "id" : "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/privateEndpoints/testPe",
+        "location" : "eastus",
+        "properties" : {
+          "provisioningState": "Succeeded",
+          "privateLinkServiceConnections": [

```

Comment:
There's another option to create a "manualPrivateLinkServiceConnections" on the private endpoint API, please replicate on all options PUT/GET